RecoParticleFlow/PFClusterProducer/plugins/PFHBHERecHitCreatorCUDA.cu

```diff
@@ -43,20 +43,14 @@ namespace PFRecHit {
     }
 
     // Initialize arrays used to store temporary values for each event
-    __global__ void initializeArrays(uint32_t nTopoArraySize,
+    __global__ void initializeArrays(uint32_t nTopoArraySize, // Previously nDenseIdsInRange, now takes detId.size() but needs work
```

Comment:
I think we just want to explain what it is. Something like:
Size of the size of arrays storing topology information
?